de.create3000.Titania.json

```diff
@@ -0,0 +1,427 @@
+{
+	"app-id": "de.create3000.titania",
```

Comment:
your app-id isn't consistent in capitalization, is it `de.create3000.titania` or `de.create3000.Titania`, whichever, everything needs to use the same way otherwise it'll cause issues and failures.

Reply:
Fixed that.